docs/pages/guides/typescript.mdx

```diff
@@ -76,7 +76,7 @@ The easiest way to get started is to initialize your new project using a TypeScr
 
 When you create new source files in your project you should use the **.ts** extension or the **.tsx** if the file includes React components.
 
-{/*
+{/\*
```

Comment:
```suggestion
{/* prettier-ignore */}
{/*
```

CI is failing on this line since Prettier is converting it to regular expression. Comments sometimes are weird in `.mdx` files because of Prettier.

Commenting out the above way does work when exporting locally (`yarn export`) with no errors.